core/src/main/java/org/elasticsearch/rest/action/cat/RestAliasAction.java

```diff
@@ -78,7 +78,8 @@ protected Table getTableWithHeader(RestRequest request) {
         return table;
     }
 
-    private Table buildTable(RestRequest request, GetAliasesResponse response) {
+    // package private for testing
+    Table buildTable(RestRequest request, GetAliasesResponse response) {
```

Comment:
Can you revert this change, I do not see any tests added that rely on the visibility being changed?